modules/experimental/src/gltf/gltf-material-parser.js

```diff
@@ -140,4 +145,61 @@ export default class GLTFMaterialParser {
       });
     }
   }
+
+  renameShaderVariable(oldName, newValue) {
+    const code = `#define ${oldName} ${newValue}\n`;
+    this.vs = code + this.vs;
+    this.fs = code + this.fs;
+  }
+
+  resolveSemanticUniform(name, {semantic}, value) {
+    // handle texture
+    if (value && value.texture) {
+      if (value.texCoord) {
+        log.warn('Multi tex-coord not supported')();
+      }
+
+      this.parseTexture(value, name);
+    } else if (semantic === 'CESIUM_RTC_MODELVIEW' || semantic === 'MODELVIEW') {
+      this.renameShaderVariable(name, 'u_ModelView');
+    } else if (semantic === 'PROJECTION') {
+      this.renameShaderVariable(name, 'u_Projection');
+    } else {
+      log.warn('Cannot resolve uniform in glTF custom technique')();
```

Comment:
You could add cases for all the available semantics and issue a different "KHR_... glTF extension: uniform semantic ${...} not implemented" warning in those cases, different from "KHR_... gltTF extionsion: uniform semantic ${...} not recognized".